blueprints/blueprint/index.js

```diff
@@ -2,20 +2,4 @@
 
 module.exports = {
   description: 'Generates a blueprint and definition.',
-
-  files() {
```

Comment:
The `blueprint` blueprint doesn't contain a `.jshintrc` file anymore, so this `files` logic was redundant.